src/main/java/sirius/tagliatelle/emitter/Emitter.java

```diff
@@ -101,4 +105,36 @@ public Position getStartOfBlock() {
      *                improved error messages).
      */
     public abstract void visitExpressions(@Nonnull Function<Position, ExpressionVisitor> visitor);
+
+    /**
+     * Emits escaped comments to the rendering context with DEBUG level
+     *
+     * @param context the render context which provides access to the local and global environment and also the target
+     *                for the generated text
+     * @param message text to output with {@link GlobalRenderContext.DebugLevel#DEBUG}
+     */
+    public void emitDebugMessage(@Nonnull LocalRenderContext context, String message) {
+        emitMessage(context, GlobalRenderContext.DebugLevel.DEBUG, message);
+    }
+
+    /**
+     * Emits escaped comments to the rendering context with TRACE level
```

Comment:
use @link for Debug level and add '.' at end of sentence

Reply:
JavaDoc updated.